bosh_openstack_cpi/lib/cloud/openstack/cloud.rb

```diff
@@ -305,6 +305,12 @@ def create_vm(agent_id, stemcell_id, resource_pool,
 
           with_openstack { server.destroy }
 
+          begin
+            wait_resource(server, [:terminated, :deleted], :state, true)
+          rescue Bosh::Clouds::CloudError => deleteServerError
+            @logger.warn("Failed to destroy server: #{deleteServerError.message}")
```

Comment:
use `#{delete_server_error.inspect}\n#{delete_server_error.backtrace.join("\n")}` to get the class name and the backtrace of the error